src/sb_pointcloud_processing/include/DBSCAN.h

```diff
@@ -46,17 +50,25 @@ class DBSCAN {
      */
     vector<unsigned int>* _neighbors;
 
-    int _min_neighbors = 5;
-    float _radius      = 5;
-
+    int _min_neighbors               = 5;
+    float _radius                    = 5;
     unsigned int _sequential_cut_off = 1000;
+    Plane _plane;
 
-  public:
     /*
      * Constructor:
      * Takes in minimum number of neighbours and radius as parameters
+     * as well as plane (XY or YZ)
```

Comment:
We should remove `(XY or YZ)` here in case the enum ever changes (because it's likely this doc comment wouldn't get updated along with the enum).